examples/lock-app/p6/README.md

```diff
@@ -0,0 +1,153 @@
+# Matter P6 Lock Example
+
+An example showing the use of Matter on the Infineon CY8CKIT-062S2-43012 board.
+
+<hr>
+
+-   [Matter P6 Lock Example](#chip-p6-lock-example)
+    -   [Introduction](#introduction)
+    -   [Building](#building)
+    -   [Flashing the Application](#flashing-the-application)
+    -   [Commissioning and cluster control](#commissioning-and-cluster-control)
+        -   [Setting up Python Controller](#setting-up-python-controller)
+        -   [Commissioning over BLE](#commissioning-over-ble)
+            -   [Notes](#notes)
+        -   [Cluster control](#cluster-control)
+
+<hr>
+
+<a name="intro"></a>
+
+## Introduction
+
+The P6 lock example provides a demonstration of a door lock control device,
+built using Matter and the Infineon Modustoolbox SDK. It can be controlled by a
+Matter controller over Wi-Fi network.
+
+The P6 device can be commissioned over Bluetooth Low Energy where the device and
+the Matter controller will exchange security information with the Rendezvous
+procedure. Wi-Fi Network credentials are then provided to the P6 device which
+will then join the network.
+
+<a name="building"></a>
+
+## Building
+
+-   Download
+    [Modustoolbox Software](https://www.cypress.com/products/modustoolbox)
```

Comment:
Is there a separate PR to add the required SDK as part of the CHIP docker image?

We would like developers to be able to test compile using the vscode image (which ends up referencing all SDKs) and have the option to add things to CI (even if a bit overloaded now)

Reply:
Added Docker vscode changes as requested. Also added script for compiling P6 lock app which can be used for CI workflows (this will be added in next PR)

Reply:
Please document the option to dowload without the 'registration' step (maybe referencing the dockerfile paths)

Reply:
Updated README.md to refer to dockerfile for software download steps